branchLayout/api/src/main/java/com/virtuslab/branchlayout/api/BranchLayout.java

```diff
@@ -30,6 +33,20 @@ private static List<IBranchLayoutEntry> collectEntriesRecursively(IBranchLayoutE
     return entryByName.get(branchName).getOrNull();
   }
 
+  @Override
+  public @Nullable IBranchLayoutEntry findDirectParentBranch(@NonNull IBranchLayoutEntry actualEntry,
```

Comment:
Hmm what's the meaning of `actualEntry` and `currentEntry`? pls add javadoc as it isn't clear

Reply:
This method won't be needed soon... I'm almost ready with https://github.com/VirtusLab/git-machete-intellij-plugin/issues/430 ;)

Reply:
Hm, so maybe I'll wait for your solution to be merged to develop. Then I will update my logic to use your solution.